Description: Harden binary by using default build flags.
Author: Doug Torrance
Last-Update: 2014-10-29

--- a/wmifs/Makefile
+++ b/wmifs/Makefile
@@ -23,10 +23,10 @@
 INSTALL_FILE    = $(INSTALL) -p -o root -g root -m 644
 
 .c.o:
-	$(CC) -c $(CFLAGS) $< -o $*.o
+	$(CC) $(CPPFLAGS) $(CFLAGS) -c $< -o $*.o
 
 wmifs: $(OBJS)
-	$(CC) -o wmifs $^ -lXext $(LIBDIR) $(LIBS)
+	$(CC) $(LDFLAGS) -o wmifs $^ -lXext $(LIBDIR) $(LIBS)
 
 all:: wmifs
 
